app/scripts/proactive/config.js

```diff
@@ -6,6 +6,7 @@ define(function () {
     
     return {
         restApiUrl: '/rest/studio',
+        execution_scheduler_restApiUrl: '/execution-scheduler/scheduled-workflows',
```

Comment:
I agree with you. But i am not sure it could be the case for us, since the service's war is packaged along with the scheduler release package, and gets deployed in the same jetty server.